english/light-fire/graham-crackers/chocolate/find-chocolate.md

```diff
@@ -1,11 +1,2 @@
 You don't find any chocolate in your jacket, but you do find some pre-fermented and dried cocoa beans and some sugar.
 
-You realize that you could make your own chocolate out of these ingredients,
-but it would probably take a while. You take a moment to weigh your options.
-
-You decide to:
-
-[Be lazy](../eat-goo/eat-goo.md) (just eat the graham crackers and marshmallow goo already)
-
-[Be patient](make-chocolate/make-chocolate.md) (try to make chocolate)
-
```

Comment:
Hmm... looks like I need to merge the upstream master into this branch again, since my other pull request adding the the story has been merged.

Initially I had 2 branches coming off the latest master that looked like this:
```
    * smores-dilemma
    | * typo-fix
    |/
    * master
```
where each was its own logical change. I submitted a separate pull request for each of them. It looks like when the first one was merged, the second became out-of-date.

Is there a better way I could have done this?